notebooks/00_preprocess_data.ipynb

```diff
@@ -0,0 +1,494 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Steps to preprocess FIADB data for use in `synthetic-knn`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is a walkthrough for getting data from the Forest Inventory and Analysis Database (FIADB) into a format that can be used in the `synthetic-knn` package. Data for FIADB is hosted at the [FIA DataMart](https://apps.fs.usda.gov/fia/datamart/datamart.html) and for this example, we will be downloading data for the state of Oregon from 2011-2020.  This constitutes a full cycle of plots (10 years).\n",
+    "\n",
+    "The first step is to download the data from the FIA DataMart.  Select `SQLite` format and select \"Oregon\" in the \"Select State/s\" dropdown.  In the \"Data Available for Download\" section, click on the \"SQLite_FIADB_OR.zip\" link to download the data.  This will download a zip file that contains the SQLite database.  Open this zipfile in an analysis directory such that the SQLite database is called \"SQLite_FIADB_OR.db\", then follow the below steps to create the required CSV files for the `synthetic-knn` package."
```

Comment:
> maybe worth adding a direct download link to https://apps.fs.usda.gov/fia/datamart/Databases/SQLite_FIADB_OR.zip as a shortcut

Great idea.  I've done this now.

> You could also add a cell with a `wget` command to directly download the file, but maybe that's not universal enough to be worth including, since it won't work for most Windows users?

This is also a great idea, but I decided that it wasn't worth the complexity making it work for Windows users.